src/services/demoRequest.service.ts

```diff
@@ -0,0 +1,37 @@
+// Dao
+import DemoRequestDAO from '@/dao/demoRequest.dao';
+import SessionDAO from '@/dao/session.dao';
+// Helpers
+import EmailHelper from '@/helpers/email.helper';
+// Formatters
+import { DemoRequestFormatter } from '@/formatters/demoRequest.formatter';
+// Typings
+import { BookDemoRequestData } from '@/typings/demoRequest';
+
+class DemoRequestService {
+  // Dao
+  private demoRequestDAO = new DemoRequestDAO();
+  private sessionDAO = new SessionDAO();
+  // Helpers
+  private emailHelper = new EmailHelper();
+  // Formatters
+  private demoRequestFormatter = new DemoRequestFormatter();
```

Comment:
not sure about this.
but according to me initialising a formatter is of no use it is consuming memory only.
it should be exported directly like utils
@HiAkshat whats your opinion

Reply:
@pratik-edu also can we implement singleton design also here ??